com.github.jkotra.unlockr.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.github.jkotra.unlockr
+runtime : org.gnome.Platform
+runtime-version : '42'
+sdk : org.gnome.Sdk
+command: unlockr
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --device=dri
+
+modules:
+  - name: qpdf
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: 'https://github.com/qpdf/qpdf'
+        tag: "release-qpdf-10.6.3"
+    cleanup:
+      - "*.a"
```

Comment:
no `/include` ? `*.la` ? pkgconfig files?

Reply:
done. ~600kb reduced.

let me know if i miss anything else. :+1:

Reply:
you could check the content of the flatpak. The build dir contain all the files.

(tbf I wish the build bot allowed listing the files)